compiler/syntax/src/res_parens.ml

```diff
@@ -383,7 +382,7 @@ let jsx_child_expr expr =
          ( Pexp_ident _ | Pexp_constant _ | Pexp_field _ | Pexp_construct _
          | Pexp_variant _ | Pexp_array _ | Pexp_pack _ | Pexp_record _
          | Pexp_extension _ | Pexp_letmodule _ | Pexp_letexception _
-         | Pexp_open _ | Pexp_sequence _ | Pexp_let _ );
+         | Pexp_open _ | Pexp_sequence _ | Pexp_let _ | Pexp_jsx_element _ );
```

Comment:
double checking: only 1 of the 3 jsx ast nodes here is correct?

Reply:
This captures all three nodes, so what exactly do you mean here?